### PR TITLE
Clarify unique Fastly credentials per environment

### LINKED ISSUE #8069: Feedback on page: /cloud/cdn/configure-fastly.html
# General issue

## Description:

https://devdocs.magento.com/cloud/cdn/configure-fastly.html#cloud-fastly-creds

```
Use the following methods to find and save the Fastly service ID and API token for your environment:

To view your Fastly credentials:
* IaaS-mounted shared directory—On Pro projects, use SSH to connect to your server and get the Fastly credentials from the /mnt/shared/fastly_tokens.txt file.
```

When Cloud Pro customers submit tickets relating to Fastly not working correctly, their issue often arises simply due to them having configured the wrong Fastly credentials in the affected environment.

## Possible solutions:

Add an alert for Pro customers under that segment that they should make sure to check the /mnt/shared/fastly_tokens.txt file in the correct environment for the Fastly credentials when configuring Fastly because **Staging and Production have separate credentials/Service IDs**.

### PR DESCRIPTION
**Pull request by meker12**
Fixes #8069 
## Purpose of this pull request

Added clarification that Fastly credentials are unique for each Cloud environment.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/cdn/configure-fastly.html

### CODE REVIEW

File: src/cloud/cdn/configure-fastly.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ After you enable and verify that Fastly works with the default settings, you can
 
 ## Get Fastly credentials {#cloud-fastly-creds}
 
-During project provisioning, Magento adds your project to the [Fastly service account]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html#fastly-service-account-and-credentials) for {{ site.data.var.ece }} and adds the Fastly account credentials to the Staging and Production environment configuration.
+During project provisioning, Magento adds your project to the [Fastly service account]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html#fastly-service-account-and-credentials) for {{ site.data.var.ece }} and Fastly account credentials to the Staging environment configuration and the Production environment configuration. Each environment has unique credentials.
```
Comment (meker12) [inline src/cloud/cdn/configure-fastly.md:32]:
```suggestion
During project provisioning, Magento adds your project to the [Fastly service account]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html#fastly-service-account-and-credentials) for {{ site.data.var.ece }} and creates Fastly account credentials for the Starter `master` and Pro Staging and Production environments. Each environment has unique credentials.
```

**Comment by BarnyShergold** [comment]
@meker12 - If you are updating this area, then I think it's also important to state that if the wrong credentials are added to an environment then doing things like flushing Fastly will action this on the environment that the credentials are for. Whilst Integration does not support Fastly in itself, it can affect another environment's Fastly.

EG Developer dumps Staging database and imports into Integration but forgets to remove the Fastly credentials. If the caches are then flushed on Integration Fastly will flush the Staging environment!

The more likely scenario is that even though the credentials are on each server, someone could mix up Production & Staging....

**Comment by meker12** [comment]
> @meker12 - If you are updating this area, then I think it's also important to state that if the wrong credentials are added to an environment then doing things like flushing Fastly will action this on the environment that the credentials are for. Whilst Integration does not support Fastly in itself, it can affect another environment's Fastly.
> 
> EG Developer dumps Staging database and imports into Integration but forgets to remove the Fastly credentials. If the caches are then flushed on Integration Fastly will flush the Staging environment!
> 
> The more likely scenario is that even though the credentials are on each server, someone could mix up Production & Staging....

I understand what you are saying here. However, if people complete the instructions in the docs for setting up Fastly for Pro environments, and use the **Test credentials** option in the Admin UI, then they will have the correct credentials configured.  On Starter, the credentials are configured on behalf of the customer using environment variables.

**Review by hguthrie — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**